libs/solid/profile/src/lib/components/detail/detail.component.scss

```diff
@@ -4,30 +4,36 @@ p {
 
   &.heading {
     padding: 0.5em;
-    font-weight: 600;
+    font-weight: 500;
+    background-color: #f9f9f9;
+    margin-bottom: 0.5em;
+    margin-top: 1em;
     //padding: 24px;
   }
 }
 
 solid-core-image {
+  margin-top: 2px;
   height: calc(100vh / 3);
 }
 
 mat-expansion-panel {
   mat-expansion-panel-header {
     mat-panel-title {
-      font-weight: bold;
+      font-weight: 500;
     }
-    //padding: 0 24px 0 0;
+  }
+  mat-expansion-panel-header.mat-expanded {
+    background-color: #f5f5f5;
   }
 }
 
 div.img-toolbar {
-  //margin-top: -6px;
-  overflow-y: scroll;
+  margin: 10px 0 10px 0;
+  //overflow-y: scroll;
+  overflow-x: auto;
```

Comment:
This is a quite different from what it was but it appears to be doing what I want to achieve (get rid of horizontal scrollbar and allow for horizontal scrolling). Any ideas about side-effects?
And why can I scroll fully to the right (until `>`) but not to the right in the thumbnail toolbar?